packages/edit-site/src/components/editor/index.js

```diff
@@ -274,6 +288,22 @@ export default function Editor( { listViewToggleElement, isLoading } ) {
 						footer={
 							shouldShowBlockBreadcrumbs && (
 								<BlockBreadcrumb
+									isHighlighted={ isEditingTemplate }
+									onBackClick={
+										isEditingTemplate
+											? () =>
+													setRenderingMode(
+														getEditorSettings()
+															.defaultRenderingMode
+													)
+											: undefined
+									}
+									backLabel={
+										isEditingTemplate
+											? contextPost?.title?.rendered ||
+											  'Back'
+											: undefined
+									}
```

Comment:
To be honest, it feels like the back button shouldn't be part of the block breadcrumb but just rendered before it. (It has nothing to do with blocks)

Also, should we do the same in the post-editor, we could write that component in the editor package